packages/salesforcedx-utils-vscode/src/helpers/paths.ts

```diff
@@ -25,6 +21,7 @@ const DEBUG = 'debug';
 const LOGS = 'logs';
 const APEX_DB = 'apex.db';
 const LWC = 'lwc';
+const SFDX_CONFIG = 'sfdx-config.json';
```

Comment:
nit: can we call this SFDX_CONFIG_FILE?  Kind of a shame we can't get that from core somewhere.